charts/barman/Chart.yaml

```
@@ -24,3 +24,8 @@ annotations:
       links:
         - name: GitHub PR
           url: https://github.com/adfinis/helm-charts/pull/905
+    - kind: change
+      description: "change barman default docker images"
+      links:
+        - name: GitHub PR
+          url: https://github.com/adfinis/helm-charts/pull/913
```

This should reference the new image and not itself. The self reference from the previous change was not a good example but didn't get caught in review.

```suggestion
    - kind: change
      description: "use image from basalam/barman-docker"
      links:
        - name: Image Source
          url: https://github.com/basalam/barman-docker
```

Also, please remove the previous "changes" entry. The way artifacthub works, the chart only needs to contain the changes done in this individual change and it will then aggregate them into a complete changelog via the artifacthub api.